.github/workflows/cd.yml

```diff
@@ -101,18 +107,13 @@ jobs:
             echo "::set-output name=CHANGED::false"
           fi
 
-      - name: release daily npm packages to npmjs.org
-        if: ${{ github.ref == 'refs/heads/dev' && startsWith(github.event.inputs.version, 'pre') }}
-        run: |
-          npx lerna publish --no-private --preid=alpha --dist-tag=alpha ${{ github.event.inputs.version }} --yes ${{ github.event.inputs.force }}
-
-      - name: release rc npm packages to npmjs.org
-        if: ${{ startsWith(github.ref, 'refs/heads/release/') && github.event.inputs.version == 'prerelease' }}
+      - name: release prerelease npm packages to npmjs.org
+        if: ${{ startsWith(github.event.inputs.version, 'pre') }}
```

Comment:
Does it mean daily release and pre release can be published from arbitrary branches? Do we need to restrict the branch like the stable release?

Reply:
I’ve already added some validations. Thanks bro.